server/src/main/java/io/crate/planner/operators/Collect.java

```diff
@@ -344,7 +344,7 @@ public FetchRewrite rewriteToFetch(TableStats tableStats, Collection<Symbol> use
             } else if (!SymbolVisitors.any(Symbols.IS_COLUMN, output)) {
                 newOutputs.add(output);
                 replacedOutputs.put(output, output);
-            } else if (SymbolVisitors.any(usedColumns::contains, output)) {
+            } else if (isOutputUsed(output, usedColumns)) {
```

Comment:
Wouldn't `SymbolVisitors.any(output::equals, usedColumns)` also work?

Reply:
Yep, it worked, thanks for the shorter version!